frontend/tests/components/profile/profile-content.test.tsx

```
@@ -0,0 +1,698 @@
+
+      expect(linkedInLink).toHaveAttribute(
+        "href",
+        "https://linkedin.com/in/johndoe",
+      );
+      expect(githubLink).toHaveAttribute("href", "https://github.com/johndoe");
+    });
+  });
+
+  describe("Statistics Panel", () => {
+    it("calculates total enrollments correctly", () => {
+      const { container } = render(
```

ESLint

[error] 'container' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)